modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/IgniteSnapshotManager.java

```diff
@@ -730,8 +732,6 @@ public static String partDeltaFileName(int partId) {
 
             locSnpTasks.clear();
 
-            snpRmtMgr.stop();
```

Comment:
Why changed ?

Reply:
`snpRmtMgr` must stop accepting new tasks in order to safely interrupt and stop `restoreCacheGrpProc`. Otherwise, the restore process may hang when the node is stopping (sometimes reproduced in the test).